mpd/events.go

```diff
@@ -0,0 +1,18 @@
+package mpd
+
+import "encoding/xml"
+
+type EventStream struct {
+	XMLName     xml.Name `xml:"EventStream"`
+	SchemeIDURI *string  `xml:"schemeIdUri,attr"`
+	Value       *string  `xml:"value,attr,omitempty"`
+	Timescale   *int64   `xml:"timescale,attr"`
+	Events      []*Event `xml:"Event,omitempty"`
```

Comment:
Since this is new code, can we do `[]Event` instead? to the best of my knowledge there isn't a reason to have slice of pointers here